5_databases/week3/queries.sql

```diff
@@ -0,0 +1,133 @@
+-- Active: 1699540080106@@0.0.0.0@3306@mealsharing
+
+    ---- Meal ----
+
+--Get all meals
+select *
+FROM meal;
+
+--Add a new meal
+INSERT INTO meal (id, title, description, location, `when`, max_reservations, price, reated_date)
+VALUES(3, "Naan", "Indian bread with garlic and butter", "Indian Royal", "2023-12-10 14:21:00", 5, "90", "2023-07-11");
```

Comment:
When inserting you most often don't want to explicitly insert the `id` as it is an `AUTO_INCREMENT` column. If you omit it, the database will automatically assign it the first available id. This way you don't have to figure out the next id to use and you also don't risk accidently assigning an id already in use.
```sql
INSERT INTO meal (title, description, location, `when`, max_reservations, price, reated_date)
VALUES("Naan", "Indian bread with garlic and butter", "Indian Royal", "2023-12-10 14:21:00", 5, "90", "2023-07-11");
```